testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -2791,6 +2791,22 @@ public void testInsertIntoNotNullColumn()
             // This is enforced by the engine and not the connector
             assertQueryFails(format("INSERT INTO %s (not_null_col, nullable_col) VALUES (NULL, 3)", table.getName()), "NULL value not allowed for NOT NULL column: not_null_col");
         }
+
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "not_null_no_cast", "(nullable_col INTEGER, not_null_col INTEGER NOT NULL)")) {
```

Comment:
Would it be possible to reuse `TestTable` declaractions?
this would speed up the test a bit

Reply:
sure I can give it a try, tough there were 3 already before my change